src/bp/core/services/stats/index.ts

```diff
@@ -0,0 +1,215 @@
+          count: await this.getEntitiesCount()
+        }
+      },
+      qna: {
+        count: await this.getQnaCount()
+      },
+      hooks: {
+        global: {
+          count: await this.getGlobalHooksCount()
+        }
+      },
+      actions: {
+        global: {
+          count: await this.getGlobalActionsCount()
+        },
+        bot: {
+          count: await this.getBotActionsCount()
+        }
+      },
+      contentElements: {
+        count: await this.getContentElementsCount()
+      },
+      server: {
+        externalUrl: process.EXTERNAL_URL,
+        botpressVersion: process.BOTPRESS_VERSION,
+        fingerprint: await this.getServerFingerprint(),
+        clusterEnabled: yn(process.CLUSTER_ENABLED, { default: false }),
+        machineUUID: await machineUUID(),
+        nodesCount: await this.jobService.getNumberOfSubscribers(),
+        os: process.platform,
+        totalMemoryBytes: os.totalmem(),
+        uptime: Math.round(process.uptime()),
+        bpfsStorage: process.BPFS_STORAGE,
+        dbType: this.database.knex.isLite ? 'sqlite' : 'postgresql'
+      },
+      license: {
+        type: process.IS_PRO_ENABLED ? 'pro' : 'ce',
+        status: await this.getLicenseStatus(),
+        isProAvailable: process.IS_PRO_AVAILABLE,
+        showPoweredBy: config.showPoweredBy
+      },
+      users: {
+        superAdmins: {
+          count: config.superAdmins.length
+        },
+        collaborators: {
+          count: await this.getCollaboratorsCount()
+        },
+        chat: {
+          count: await this.userRepository.getUserCount()
+        }
+      },
+      auth: {
+        strategies: {
+          count: Object.keys(config.authStrategies).length
+        }
+      }
+    }
+  }
+
+  private async getBotsCount(): Promise<number> {
+    return (await this.botService.getBotsIds()).length
+  }
+
+  private async getWorkspacesCount(): Promise<number> {
+    return (await this.workspaceService.getWorkspaces()).length
+  }
+
+  private async getPipelineStagesCount(): Promise<number> {
+    const workspaces = await this.workspaceService.getWorkspaces()
+    return workspaces.reduce((acc, workspace) => {
+      return acc + workspace.pipeline.length
+    }, 0)
+  }
+
+  private async getFlowCount(): Promise<number> {
+    return (await this.ghostService.bots().directoryListing('/', '*/flows/*.flow.json', '*/flows/error.flow.json'))
+      .length
+  }
+
+  private async getIntentsCount(): Promise<number> {
+    return (await this.ghostService.bots().directoryListing('/', '*/intents/*')).length
+  }
+
+  private async getEntitiesCount(): Promise<number> {
+    return (await this.ghostService.bots().directoryListing('/', '*/entities/*')).length
+  }
+
+  private async getQnaCount(): Promise<number> {
+    return (await this.ghostService.bots().directoryListing('/', '*/qna/*')).length
+  }
+
+  private async getContentElementsCount(): Promise<number> {
+    return this.cmsService.countContentElements()
+  }
```

Comment:
I think we could use directly this.cmsService in the getStats instead of that wrapper